### PR TITLE
[Part 2] Migrate Topics API to Netlify functions

### PR DESCRIPTION
**Pull request by tuttiq**
### References / ticket
- Part 2 of issue #15 
- Follow-up from PR https://github.com/WWCodeTokyo/speak-her-db/pull/143 **(already merged!)**
- Also solves issue https://github.com/WWCodeTokyo/speak-her-db/issues/144

### Proposed Changes

- [x] Continuation of the REST API hosted on Netlify Lambda functions under `src/lambda/` ([details here](https://github.com/WWCodeTokyo/speak-her-db/pull/143))
    - [x] For this PR, I created the `topics.js` function/endpoint
- [x] Added `getTopics` to the API service. It can be used across the app like this:
```javascript
import api from '@/services/api'; 
api.getLocations(successCallback, errorCallback);
```
- [x] Migrate `Topic` requests across the app from the Airtable plugin to the new API service
    - [x] Fetch Topics on `components/TopicsInput.vue`
    - [x] Fetch Topics on `views/FindSpeaker.vue`

- [x] Moved the request for Topics from `components/CardFooter` to `views/FindSpeaker` and passing the `topicList` down as props to `SpeakerCard` > `CardFooter` to fix this [N+1 query issue](https://github.com/WWCodeTokyo/speak-her-db/issues/144)

### Testing Plan

- Open [this deploy preview](https://deploy-preview-145--speak-her-db.netlify.app/)
- When the site is loading, open the network inspector and check that there's only 1 `topics` request now instead of many (there were ~70 before)
- On the network inspector click on the single `topics` request
- Request headers now show the endpoint is `/.netlify/functions/topics` and there's no authorization key on the request headers 🔒 

Alternatively, run it locally:
- Pull this branch locally, run `yarn install` to install the new dependencies
- If your connection to Airtable is already setup locally as per README instructions, just run the app with `yarn serve`
- Open `localhost:8080` on your browser and follow the steps above

### CODE REVIEW

**Comment by tuttiq** [comment]
@ann-kilzer @yanarchy Ready for review but needs https://github.com/WWCodeTokyo/speak-her-db/pull/143 to be merged first.

**Comment by tuttiq** [comment]
@ann-kilzer this is ready for review now

File: web/src/components/speaker/SpeakerCard.vue [hunk 1]
```diff
@@ -35,12 +35,15 @@
               >
                 {{ $t('findSpeaker.pronouns', [pronouns]) }}
               </div>
-              <p class="pa-0 ma-0">{{ speaker.get('speaker_bio') }}</p>
+              <p class="pa-0 ma-0">
```
Comment (ann-kilzer) [inline web/src/components/speaker/SpeakerCard.vue:38]: Do you have prettier or some other formatter moving things here? (Should we all use prettier? I haven't used it much and have mostly relied on eslint)

**Review by ann-kilzer — APPROVED** [review]
LGTM 🌈 ☔

File: web/src/components/speaker/SpeakerCard.vue (hunk 1 above)
Comment (tuttiq) [reply web/src/components/speaker/SpeakerCard.vue:38]: I don't think I have prettier, I have been running `yarn lint --fix` which uses the project configuration and not the editor one (I disabled auto-formatting on my editor) 🤔

**Pull request merged**